### DISCUSSION TITLE
貼り付けたコードを記事内で簡単に参照できる

### DISCUSSION THREAD

**Discussion by kokosabu** [increments/qiita-discussions#24] [要望・アイデア・不具合]
少し長めのコードを説明する記事の場合、始めか最後にコード全体を載せた上で、ポイントとなるような行を取り上げて詳しく説明するスタイルの記事があると思います。

そんな時、貼り付けた全体コードを、記事内で一部抜粋して表示する機能があると便利ではないでしょうか。

**Comment by murakami** [comment]
なるほど。例えば、行番号が表示されると、場所の説明が便利ですよね。

**Comment by kyntk** [comment]
@kokosabu @murakami ご意見ありがとうございます。
コードブロックの行数を表示したり、参照する処理については、すぐには実装が難しそうなのですが、長めのコードを説明する記事を読みやすくする仕方については検討させていただきます!
